src/BlockTypes/AbstractBlock.php

```diff
@@ -175,8 +175,9 @@ protected function register_chunk_translations( $chunks ) {
 	 * Generate an array of chunks paths for loading translation.
 	 */
 	protected function get_chunks_paths() {
-		foreach ( glob( \Automattic\WooCommerce\Blocks\Package::get_path() . "build/{$this->chunks_folder}/*-frontend.js" ) as $block_name ) {
-			$blocks[] = "{$this->chunks_folder}/" . basename( $block_name );
+		$build_path = \Automattic\WooCommerce\Blocks\Package::get_path() . 'build/';
+		foreach ( new \RecursiveIteratorIterator( new \RecursiveDirectoryIterator( $build_path . $this->chunks_folder ) ) as $block_name ) {
+			$blocks[] = str_replace( $build_path, '', $block_name );
```

Comment:
h/t for @dinhtungdu
I switched from glob to `RecursiveDirectoryIterator` because glob didn't support nested folders and I didn't feel like hacking it.

`RecursiveDirectoryIterator` will get you all folders, and `RecursiveIteratorIterator` will iterate over all iterations.